cmd/run.go

```diff
@@ -15,16 +16,21 @@ import (
 
 const defaultFilename = "/var/log/nginx/mirrors/access_json.log"
 
-func filenameFromArgs(args []string) string {
+func filenamesFromArgs(args []string) []string {
 	if len(args) == 0 {
-		return defaultFilename
+		return []string{defaultFilename}
 	}
-	return args[0]
+	return args
 }
 
 func runWithConfig(cmd *cobra.Command, args []string, config analyze.AnalyzerConfig) error {
-	filename := filenameFromArgs(args)
-	fmt.Fprintln(cmd.ErrOrStderr(), "Using log file:", filename)
+	filenames := filenamesFromArgs(args)
+	// Allow multiple files only when ananlyzing and NOT daemonizing
+	if !(config.Analyze && !config.Daemon) && len(filenames) != 1 {
```

Comment:
I think that `!config.Analyze` is enough, as there are only 3 modes:
- follow (run): both are false
- analyze: Analyze = true
- daemon: Daemon = true